tests/blend.py

```diff
@@ -2,24 +2,34 @@
 
 import pytest
 import numpy as np
-from minerva_lib.blend import to_f32
-from minerva_lib.blend import f32_to_bgr
-from minerva_lib.blend import linear_bgr
+from minerva_lib.blend import make_rgb
+from minerva_lib.blend import clip_image
+from minerva_lib.blend import linear_rgb
+
+
+@pytest.fixture
+def full_u8():
+    return int(255)
+
+
+@pytest.fixture
+def full_u16():
+    return int(65535)
 
 
 @pytest.fixture
 def range_all():
-    return np.float32([0, 1])
+    return np.array([0, 1], dtype=np.float32)
 
 
 @pytest.fixture
 def range_high():
-    return np.float32([0.5, 1])
+    return np.array([0.5, 1], dtype=np.float32)
 
 
 @pytest.fixture
-def range_low():
-    return np.float32([0, 256. / 65535.])
+def range_low(full_u8, full_u16):
+    return np.array([0, full_u8 / full_u16], dtype=np.float32)
```

Comment:
What's the interpretation of `full_u8 / full_u16`?

Reply:
It's the `f32 [0, 1]` representation of `255` from a uint16 image. This actual value of `~0.0038911` is the closest possible input value to `~0.0039216`, which is the `f32 [0, 1]` representation of the lowest nonzero value of `1` in a uint8 representation of the output image. 

This `full_u8 / full_u16` is equal to the "medium" value of `f32_low_med_high`, which is the `f32 [0, 1]` representation of `u16_low_med_high`, which is `[(2 ** n) - 1 for n in (0, 8, 16)]`. This `u16_low_med_high` is chosen to resolve to `[0, 1, 255]` in a uint8 representation of an output  image.

These exact values were very important for testing rounding behavior, but the exact values have become much less important now that we're not rounding the output to `uint8`.

Reply:
Updated to use 12345 / 65535

This now can be any fraction with a denominator of uint16.max